crates/bevy_asset/src/asset_server.rs

```diff
@@ -499,9 +507,11 @@ impl AssetServer {
                             .get_or_insert_with(|| self.server.asset_sources.write());
                         if let Some(source_info) = asset_sources.get_mut(&id.source_path_id()) {
                             source_info.committed_assets.remove(&id.label_id());
```

Comment:
A second question for people, if `committed_assets.len() == 0`, should we remove the entirety of the `SourceInfo` struct?